test/mostCommon.js

```diff
@@ -0,0 +1,13 @@
+const solution = require('../solutions/mostCommon.js');
+
+const mostComm = (ar, result)=>{
+  if(solution(ar) == result){
+    console.log('Most common number');
+  }else{
+    console.log('Not most common number');
```

Comment:
I think you should do `console.log(Not most common number. The answer is: ${result})` so that you can let whoever running the test know the acutual common number.